modules/mcuboot/CMakeLists.txt

```
@@ -127,6 +132,7 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       # IMAGE_MAGIC into the image trailer.
       ${sign_cmd}
       ${sign_dependencies}
+      ${rom_fixed_option}
```

minor nit: it can be risky to use a variable inside a fuction where it has not been ensured whether such variable has been defined or not.

Reason for that is that in case `rom_fixed_option` already exists in the caller's scope, then that value will actually be used, which often come as a surprise and lead to hard-to-debug issues.

See other comment for proposed handling.
```suggestion
      ${SIGN_ARG_ROM_FIXED}
```